testcontainers-gitserver/src/main/java/com/github/sparsick/testcontainers/gitserver/http/GitHttpServerContainer.java

```diff
@@ -55,20 +55,20 @@ private static Consumer<DockerfileBuilder> dockerfileBuilder(DockerImageName doc
                     .from(dockerImageName.toString())
                     .run("apk add --update nginx && " +
                             checkUpdateGit(dockerImageName) +
-                            "apk add --update git-daemon &&" +
-                            "apk add --update fcgiwrap &&" +
+                            "apk add --update git git-daemon && " +
```

Comment:
Please move the installation of the package git to the method checkUpdateGit to have this in one place. Maybe the name of the method is now wrong. But that can be renamed.

Reply:
Updated as requested.